crawl4ai/cli.py

```
@@ -31,8 +31,10 @@
     BFSDeepCrawlStrategy,
     DFSDeepCrawlStrategy,
     BestFirstCrawlingStrategy,
+    LLMConfig,
```

Ruff (0.12.2)

Redefinition of unused `LLMConfig` from line 30

Remove definition: `LLMConfig`

(F811)